### PR TITLE
[WIP][IMP] added new fields to soport panama localization in checkout form #HU 547,#CA 1

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

**Comment by gquilarque** [comment]
Hello @hugho-ad ,

I see that you added new fields with data:

-Neighborhood
-Township
-State / Province
-District

and data for field:
-City

these new features are fine.

But, I can't see the #CA1 of #HU 547 is not ready yet in this PR.
Also missing the translate for these new label fields.

I review these features in this link: https://github.com/Vauxoo/yoytec/pull/198

**Comment by hugho-ad** [comment]
@gquilarque This PR has in the messages description [WIP]- working in progress, I'm working in the back end at the same time in front-end, I'll notify you when this is ready

Regards!

**Comment by gquilarque** [comment]
Ok @hugho-ad, thanks for your notice.

File: website_checkout_validations/controllers/main.py [hunk 1]
```diff
@@ -23,6 +23,24 @@ def checkout_values(self, data=None):
         cities = orm_city.browse(cr, SUPERUSER_ID, city_ids, context)
         states_ids = state_orm.search(cr, SUPERUSER_ID, [], context=context)
         states = state_orm.browse(cr, SUPERUSER_ID, states_ids, context)
+
+        distroct_orm = registry.get('res.country.state.district')
```
Comment (nhomar) [inline website_checkout_validations/controllers/main.py]: bad variable name

**Comment by oscarolar** [comment]
Please Merge this PR, now it deletes the module website_checkout_validations moved to yoytec repo as yoytec_checkout https://github.com/Vauxoo/yoytec/pull/225

**Comment by nhomar** [comment]
Merged en other PR #494 because the feature was setted privatelly due ti difficulties to make it agnostic to l10n_pa

**Pull request closed without merging**